app/sample/admin.py

```diff
@@ -0,0 +1,4 @@
+from django.contrib import admin
+from .models import SampleModel
```

Comment:
Use absolute imports and follow pep-8 convention for it.

Refer to this documentation link -> https://peps.python.org/pep-0008/#imports